google-cloud-firestore/src/main/java/com/google/cloud/firestore/BulkWriterOptions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.firestore;
+
+import javax.annotation.Nonnull;
+
+/** Options used to disable request throttling in BulkWriter. */
+public class BulkWriterOptions {
+  private final boolean disableThrottling;
+
+  BulkWriterOptions(boolean disableThrottling) {
```

Comment:
I noticed that all the other options objects have private constructors and expose `create()` instead. Should I do this here as well?

Reply:
The idea behind these classes is to offer "named arguments". In this case, we would want to have a static factory method that is called "disableThrottling" or "withThrottlingDisabled". I would suggest to look at SetOptions rather than at TransactionOptions for inspiration :)

Reply:
Not yet.